spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'board'
+
+# rubocop:disable Lint/UselessAssignment
+describe 'Game' do
+  g = GameBoard.new
```

Comment:
-Kindly use Rspec semantic `subject` to indicate that `g` is the object being tested